kong/pdk/log.lua

```diff
@@ -817,6 +817,13 @@ do
       -- the nginx doc: http://nginx.org/en/docs/http/ngx_http_upstream_module.html#upstream_status
       local upstream_status = var.upstream_status or ""
 
+      local response_source = okong.response.get_source(ongx.ctx)
+      if response_source == "exit" or response_source == "error" then
+        response_source = "kong"
+      elseif response_source == "service" then
+        response_source = "upstream"
+      end
```

Comment:
so `okong.response.get_source(ongx.ctx)` only have 3 values , `exit`, `error` or `service`?

Reply:
correct. 

https://github.com/Kong/kong/blob/6077171c2ca697322ed562335a6aff10a390ac52/kong/pdk/response.lua#L345-L364

Reply:
I prefer to use a constant table to contain these three values to prevent if someone change `function _RESPONSE.get_source(ctx) ` but forget to change it here.

Reply:
I don't have any strong opinions on this. I feel replacing the if conditions with a constant table may be not completed. The vars defined in `_RESPONSE.get_source` function should also be changed. Please ref the latest commit to get what I'm saying.

Reply:
Thank you @gszr. @tzssangglass Checkout the latest commit to see if it can address your concern. If yes. please resolve this comment. Otherwise, open a ticket then we can track it.